gremlin-python/src/main/python/gremlin_python/driver/driver_remote_connection.py

```diff
@@ -106,22 +108,26 @@ def cb(f):
     def is_closed(self):
         return self._client.is_closed()
 
-    def commit(self):
-        log.info("Submitting commit graph operation.")
-        return self._client.submit(Bytecode.GraphOp.commit())
-
-    def rollback(self):
-        log.info("Submitting rollback graph operation.")
-        return self._client.submit(Bytecode.GraphOp.rollback())
+    # TODO remove or update once HTTP transaction is implemented
+    # def commit(self):
+    #     log.info("Submitting commit graph operation.")
+    #     return self._client.submit(Bytecode.GraphOp.commit())
+    #
+    # def rollback(self):
+    #     log.info("Submitting rollback graph operation.")
+    #     return self._client.submit(Bytecode.GraphOp.rollback())
 
     @staticmethod
-    def _extract_request_options(bytecode):
-        options_strategy = next((x for x in bytecode.source_instructions
-                                 if x[0] == "withStrategies" and type(x[1]) is OptionsStrategy), None)
+    def extract_request_options(gremlin_lang):
```

Comment:
Could you add test for `g.with("evaluationTimeout", 1000).with("batchSize", 100).V()` to verify that more than one options_strategy is processed correctly?

Reply:
Added in drc test